adsingestp/parsers/jats.py

```diff
@@ -899,9 +899,10 @@ def citation_context(
         :param text: text of fulltext XML to parse
         :param bsparser: parser to use with BeautifulSoup
         :param input_bibcode: string, bibcode of input XML, if known # TODO do I need this? do I need to resolve and return the paper's own bibcode?
-        :param num_char: integer, check that citation paragraph is at least this long; if it's shorter, return the
+        :param num_char: integer, check that citation paragraph is at least this long; if it's shorter, return the. Note if output is BeautifulSoup oblects set num_char to 1
             paragraphs before and after the citing paragraph as well
         :param resolve_refs: boolean, set to True to convert reference IDs to bibcodes # TODO this isn't implemented yet
+        :param text_output: boolean, set to True to output citation context as a string, or False to output citation context as a beautifulSoup object
```

Comment:
See my top-level comments - update this param text if the output format changes as suggested.

Reply:
Updated.